src/Components/test/testassets/TestServer/Pages/_ServerHost.cshtml

```diff
@@ -0,0 +1,39 @@
+@page ""
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
What's motivating us to add a new entry point here?

Reply:
We can't render server-side components unless we have some server-side code (razor page) as the entry point.